_posts/2016-08-14-three-pillars-of-continuous-delivery.md

```diff
@@ -18,30 +18,30 @@ tags:
+**Make it faster**  Many different things can make your process slower. One such problem is how your processes are organised. In many cases in my previous projects the original version of the process required the individual involvement of many people in the organisation. Sometimes you don't have the right access rights to your infrastructure and deployments are impossible. Try to improve or change these elements in your organisation. The automation of your process should not only include the solutions for technical problems, but it should also improve non-technical aspects of the process.
 
 ### Team Effort
 
-**Do it together.** In my opinion the responsibility for continuous delivery should be shared by the team. By the team I mean all people involved in the project. Not only software developers, not only system administrator, we should also involve business stakeholders. Understanding continuous deployment, by all members of the team, strengthens knowledge about the software and improves cooperation. Our software is no longer just lines of code, it is also place where this software lives.
+**Do it together** In my opinion the responsibility for continuous delivery should be shared by the team. By the team I mean all people involved in the project. Not only software developers, not only system administrators, we should also involve business stakeholders. Understanding continuous deployment, by all members of the team, strengthens knowledge about the software and improves cooperation. Our software is no longer just lines of code, it is also the place where this software lives.
 
-**No Silo Culture.** Every technical person in the team should be able to start, improve and fix the continuous delivery process. I understand that this can be a challenge and many failures can appear, but only including people make your continuous delivery useful. Without the engagement of your whole team you create an invisible wall between the software development and its deployment. This can ruin all efforts of introducing automation, because your process will not be aligned with changes taking place in the software.
+**No Silo Culture** Every technical person in the team should be able to start, improve and fix the continuous delivery process. I understand that this can be a challenge and many failures can appear, but only including people make your continuous delivery useful. Without the engagement of your whole team you create an invisible wall between the software development and its deployment. This can ruin all efforts of introducing automation, because your process will not be aligned with changes taking place in code.
 
 ### Just three?
 
 You may ask why just three. For me, you cannot start doing any good continuous delivery without thinking about these three elements. These are the parts which I would like to have during when I start introducing any automation in my projects. They make all other ingredients of continuous delivery easier to add.
```

Comment:
For me,...

...these are the parts I would like to `have when` I start introducing any automation in my projects.